radicale/app/post.py

```diff
@@ -0,0 +1,5 @@
+class ApplicationPostMixin:
+    def do_POST(self, environ, base_prefix, path, user):
+        """Manage POST request."""
+        if path == "/.web" or path.startswith("/.web/"):
+            return self._web.post(environ, base_prefix, path, user)
```

Comment:
The method returns `None` for paths that do not belong to the web plugin. It should return METHOD_NOT_ALLOWED.